test/week_test.js

```diff
@@ -57,6 +57,44 @@ describe('Week', () => {
     assert(day.prop('day').isSame(dayClicked, 'day'))
   })
 
+  it('should call the provided onWeekSelect function', () => {
+    let firstDayReceived = null
+    let weekNumberReceived = null
+
+    function onWeekClick (day, newWeekNumber) {
```

Comment:
this two expectations are orthogonal so should be in separate tests

(this applies to the 'should set the week number with the provided formatWeekNumber function' test too)